README.md

```diff
@@ -1,73 +1,64 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# First Capstone
 
-> Description the project.
+> This project contains the information and events for First Capstone.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Javascript
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link]()
```

Comment:
- [ ] It would be better to deploy the website with GitHub pages and add the link here